### PR TITLE
Feature/0.9.1 modifications gdev 1447

### PR DESCRIPTION
**Pull request by sbilge**
## Proposed Changes

  - target_regions is set to optional
  - has workflow is set to optional
  - data_use_conditions class is removed, permission and modifier introduced
  - artifacts are re-generated 
  - closes GDEV 1447
  - schema version is set to 0.9.1

### CODE REVIEW

File: src/schema/ghga.yaml [hunk 1]
```diff
@@ -951,7 +951,7 @@ classes:
         description: >-
           One or more Workflow entities associated with this Analysis.
         range: workflow
-        required: true
+        required: false
```
Comment (galinatremper) [inline src/schema/ghga.yaml:954]: What about reference_chromosome (Line 935). It is required, but it was not filled in the JSON of Marc?

File: src/schema/ghga.yaml [hunk 2]
```diff
@@ -511,7 +511,7 @@ classes:
         description: >-
           One or more attributes that further characterizes this Library Preparation Protocol.
       target regions:
-        required: true
+        required: false
```
Comment (galinatremper) [inline src/schema/ghga.yaml:514]: General remark: Don't forget to update the version of the schema on the top of the file.

**Review by galinatremper — CHANGES_REQUESTED** [review]

File: src/schema/ghga.yaml (hunk 1 above)
Comment (sbilge) [reply src/schema/ghga.yaml:954]: I am not sure if we should change that. It was not provided but the implementation team has not changed it to optional for this release.

**Review by galinatremper — COMMENTED** [review]
Looks good, I'll check the models in MTS and MRS and approve the PR.

**Review by galinatremper — COMMENTED** [review]
Unfortunately, no creation models were generated...

File: src/schema/ghga.yaml [hunk 3]
```diff
@@ -1165,13 +1148,19 @@ classes:
         required: true
         inlined: true
         inlined_as_list: false
-      has data use condition:
+      has data use permission:
+        description: >-
+          Data use permission associated with a policy. Typically one or more terms from DUO
+          and should be descendants of 'DUO:0000001 data use permission'.
+        required: true
```
Comment (galinatremper) [inline src/schema/ghga.yaml:1155]: Data Use Permission is marked as optional in the spreadsheet (0.9.1), but here it is a required field...

**Review by galinatremper — CHANGES_REQUESTED** [review]

File: src/schema/ghga.yaml (hunk 3 above)
Comment (sbilge) [reply src/schema/ghga.yaml:1155]: We changed the spreadsheet field to mandatory.

**Review by galinatremper — APPROVED** [review]

**Pull request merged**